docs/howto/enabling-logging/index.adoc

```diff
@@ -0,0 +1,35 @@
+
+[#enabling-logging_{context}]
+= Enabling logging
+
+To enable logging, add the Logging fraction to the POM file of your application.
```

Comment:
Actually logging is always added :-) I mean -- it's a dependency of all fractions, so the moment you add any fraction, logging is added transitively.

Reply:
You could change this to:
> To enable logging, verify the Logging fraction has been added to the POM file of your application.